docs/integrations/working-with-custom-scripts/custom-script-samples.md

```diff
@@ -38,74 +38,85 @@ Please be aware that this custom script affects any step that comes after. There
 
 :::
 
-### Deploying Apps to Firebase App Distribution
+### Pipeline Break on Low Test Coverage
 
-Appcircle Testing Distribution provides an integrated and automated enterprise-grade solution for distributing apps to the testers, but if you want to use other solutions for app distribution, you can do so with custom scripts. You can use the following script below to deploy apps to Firebase App Distribution automatically from the Appcircle Build module.
+This document provides a sample custom script written in Ruby that can be integrated into your CI/CD pipeline to enforce a minimum test coverage threshold. The script is designed to break the pipeline if the covered test result falls below a specified percentage.
 
-- The binary to be deployed can be obtained with the related environment variable. `AC_EXPORT_DIR` and the binary path.
-- `FIREBASE_TOKEN` must be obtained through a local console. Please follow the [instructions here](https://firebase.google.com/docs/cli#cli-ci-systems) to set up the Firebase CLI locally and then you can request a token with the `firebase login:ci `command.
-- `FIREBASE_APP_ID` can be obtained from the Firebase Dashboard under the settings screen:
+:::caution
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/image (133).png' />
+Please note that this custom script must be placed after the [**Test Reports**](http://localhost:3000/workflows/android-specific-workflow-steps#test-reports-for-android) step in the workflow..
 
-The sample script is as follows:
+:::
 
-```bash
-#!/usr/bin/env bash
-#
-# Deploy a binary to Firebase App Distribution
-#
-curl -sL firebase.tools | bash
+```ruby
+require 'json'
 
-firebase appdistribution:distribute $AC_EXPORT_DIR/Runner.ipa --app $FIREBASE_APP_ID --release-notes "Release Notes..." --token $FIREBASE_TOKEN --groups "testers"
-```
+def env_has_key(key)
+    !ENV[key].nil? && ENV[key] != '' ? ENV[key] : abort("Missing #{key}.")
+end
 
-You may also use our Firebase App Distribution Component for this process.
+output_dir = env_has_key('AC_OUTPUT_DIR')
 
-https://github.com/appcircleio/appcircle-firebase-distribution-component
+def read_json_file(file_path)
+  JSON.parse(File.read(file_path))
+end
 
-### Send email notification when a build is complete
+def extract_line_coverage(json_data)
+  json_data['coverage']['lineCoverage']
+end
 
-You can send notification emails when your build is complete so that you or your teammates will know the build status.
+begin
+file_path = "#{output_dir}/test_results.json" 
```

Comment:
'test_result_file_path' instead of 'file_path'.

It is being used as 'test_result_file_path' in the line 92